.github/workflows/ci.yml

```diff
@@ -0,0 +1,114 @@
+# Used as inspiration: https://github.com/mvdan/github-actions-golang
+
+name: Cross-Platform
+
+on:
+  push:
+    branches: [ v2 ]
+  pull_request:
+    branches: [ v2 ]
+
+jobs:
+  test:
+    strategy:
+      # Default is true, cancels jobs for other platforms if one fails
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+        go-version: [ 1.14.x ]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ matrix.go-version }}
+
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Install test and coverage analysis tools
```

Comment:
Those were needed because Azure uses JUnit XML format for test result reporting. I haven't found anything as straightforward for Github Actions. There's a workaround described here:
https://github.community/t5/GitHub-Actions/Publishing-Test-Results/m-p/37236/highlight/true#M2860

Seems like a bit of a hassle though. Let's research more. Otherwise, we'll have to pull more tooling, unfortunately.